app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    result = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            result += 1
+    return result
```

Comment:
try to write it in one line 
use `count`
```
string_ = "support"
string_.count("p") ->  2
```

Reply:
I apologize, but I didn't fully understand how you see my code changes.
Should I just push the changes to Git and hit resolve conversation, or should I use re-reques review?
I accidentally pressed re-reques review, so I apologize if I caused any inconvenience.